### PR TITLE
Make @Requires repeatable (#1030)

### PR DESCRIPTION
**Pull request by Vampire**
This change is [Reviewable](https://reviewable.io/reviews/spockframework/spock/1192)

### CODE REVIEW

File: docs/extensions.adoc [hunk 1]
```diff
@@ -137,15 +137,21 @@ is checked and it is decided whether to abort the specific iteration or not.
...
 ----
-@Requires({ os.windows })
-def "I'll only run on Windows"() { ... }
+include::{sourcedir}/extension/RequiresDocSpec.groovy[tag=example-a]
 ----
 
 `Requires` works exactly like `IgnoreIf`, except that the predicate is inverted. In general, it is preferable
 to state the conditions under which a method gets executed, rather than the conditions under which it gets ignored.
 
+If multiple `@Requires` annotations are present, the annotated element is skipped if either of the
```
Comment (leonard84) [inline docs/extensions.adoc:148]: any

File: docs/extensions.adoc [hunk 2]
```diff
@@ -137,15 +137,21 @@ is checked and it is decided whether to abort the specific iteration or not.
...
+conditions evaluates to `false`:
+
```
Comment (leonard84) [inline docs/extensions.adoc:150]: Should we mention that this is basically combining the conditions with `AND`?

Comment (Vampire) [reply docs/extensions.adoc:150]: I thought that is clear enough from text and example, but I added a half-sentence.

Comment (Vampire) [reply docs/extensions.adoc:150]: I added the same half-sentence for `@IgnoreIf`

**Pull request merged**